shared/elements/gaia_switch/style.css

```diff
@@ -15,7 +15,7 @@ gaia-switch {
   font-size: 1.9rem;
   color: var(--label-color);
   padding: 1rem 0;
-  line-height: 3rem;
+  line-height: 2.6rem;
```

Comment:
Ah I see. I did this because it looks pretty bad when the label overflows onto the second line (see Settings->Browsing Privacy->Private Browsing for an example. But I can change this back for consistency.